### DISCUSSION TITLE
probes.monitoring.coreos.com is forbidden

### DISCUSSION THREAD

**Discussion by gupf0719** [prometheus-operator/kube-prometheus#663] [General] [answered]
**Environment**

```
Darwin 10.15.6 
minikube v1.12.2
Docker 19.03.12 

```

* Prometheus Operator version:

    `v0.41.0 `

* Kubernetes version information:

    `Kubernetes v1.18.6`

* Kubernetes cluster kind:

    insert how you created your cluster: minikube

* Prometheus Operator Logs:

```
 pkg/mod/k8s.io/client-go@v0.18.3/tools/cache/reflector.go:125: Failed to list *v1.Probe: probes.monitoring.coreos.com is forbidden: User "system:serviceaccount:monitoring:prometheus-operator" cannot list resource "probes" in API group "monitoring.coreos.com" at the cluster scope
```

**Comment by s-urbaniak** [comment, marked as answer]
It seems your setup does not have the proper cluster role definition as per [1]. Maybe reapplying `setup/` manifests will do the trick.

[1] https://github.com/prometheus-operator/kube-prometheus/blob/44060e5f1940d5f2dc7cb663b3123726fc4ae579/manifests/setup/prometheus-operator-clusterRole.yaml#L21
